lib/strategy/EnrichIntegrationFromRemoteConfigStrategy.js

```diff
@@ -176,7 +177,7 @@ EnrichIntegrationFromRemoteConfigStrategy.prototype._enrichWithDirectoryPolicies
     extend(config, {
       web: {
         forgotPassword: {
-          enabled: config.web.changePassword.enabled = isEnabled(directory.passwordPolicy.resetEmailStatus)
```

Comment:
Is this change intentional? That `config.web.changePassword.enabled` is not set to whatever `config.web.forgotPassword.enabled` is set to anymore?